docs/source/integrations/databricks_development_workflow.md

```diff
@@ -0,0 +1,324 @@
+# Databricks development workflow
+
+This guide demonstrates a development workflow for Kedro projects on Databricks using Databricks Repos. This hybrid workflow involves developing your Kedro project using a local Integrated Development Environment (IDE) and Databricks notebooks for testing purposes. This guide demonstrates this workflow using an example based on the Kedro PySpark Iris starter.
+This workflow has several advantages, enabling:
+
+- Use of powerful development features offered by an IDE not available on Databricks notebooks.
+- Version control of your project.
+- Multiuser development.
+
+An IDE offers benefits for development:
+
+- Auto-completion and suggestions for code, improving your development speed and accuracy.
+- Linters like Pylint or Flake8 can be integrated to catch potential issues in your code.
+- Static type checkers like Mypy can check types in your code, helping to identify potential type-related issues early in the development process.
+
+To set up these features, look for instructions specific to your IDE (for instance, [VS Code](https://code.visualstudio.com/docs/python/linting)).
+
+## Prerequisites
+
+- New or existing [AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) with administrative privileges.
+- Active [Databricks deployment](https://docs.databricks.com/getting-started/index.html) on AWS (Databricks Community Edition won't suffice as it doesn't allow you to provision personal tokens).
+- A [Databricks cluster](https://docs.databricks.com/clusters/configure.html) configured with a recent version (>= 11.3 is recommended) of the Databricks runtime.
+- [Conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) on your local machine.
```

Comment:
Does it have to be conda or is any virtual environment okay? And is this essential for this workflow or more of a best practice in general?

Reply:
I think it would be good to explain exactly that to the user, so they know why they need `conda` and could potentially decide to use another virtual environment if they know it has the right requirements.